README.md

```diff
@@ -22,12 +22,12 @@ Ruby on Rails applications.
 
 ## Key features
 
-* Heroku ready - Review apps, staging and production
-* Uses [devise](https://github.com/plataformatec/devise)
-* Uses [Heroku CI](https://heroku.com)
-* Uses [rspec](https://github.com/rspec/rspec-rails)
-* Uses [simplecov](https://github.com/colszowka/simplecov)
-* Uses [rubocop](https://github.com/bbatsov/rubocop)
+- Heroku ready - Review apps, staging and production
+- Uses [devise](https://github.com/plataformatec/devise)
+- Uses [Heroku CI](https://heroku.com)
+- Uses [rspec](https://github.com/rspec/rspec-rails)
+- Uses [simplecov](https://github.com/colszowka/simplecov)
+- Uses [rubocop](https://github.com/bbatsov/rubocop)
```

Comment:
unrelated changes?

Reply:
I tried my best to not include vs code's automatic formatting... It clearly wasn't enough!